hub/core/chunk_engine.py

```diff
@@ -198,6 +198,10 @@ def _append_bytes(self, buffer: memoryview, shape: Tuple[int], dtype: np.dtype):
 
         self.chunk_id_encoder.register_samples_to_last_chunk_id(num_samples)
 
+        last_chunk_name = self.chunk_id_encoder.get_name_for_chunk(-1)
+        last_chunk_key = get_chunk_key(self.key, last_chunk_name)
+        self.cache[last_chunk_key] = self.last_chunk
```

Comment:
can you wrap this duplicate code in a function and then call it from the 2 places

Reply:
yep done